From: Michael Froman <mfroman@mozilla.com>
Date: Thu, 2 Sep 2021 13:22:00 -0500
Subject: Bug 1654112 - fixes for hybrid build. r=ng

- adds missing includes in several places
- makes dom/media/webrtc/jsapi unified-only

Differential Revision: https://phabricator.services.mozilla.com/D124499
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/1965757ee924d49c32eab76d1a3dfa77f9eadcf4
---
 audio/channel_receive.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/audio/channel_receive.h b/audio/channel_receive.h
index dd3ca1af83..1ad3be781b 100644
--- a/audio/channel_receive.h
+++ b/audio/channel_receive.h
@@ -28,6 +28,7 @@
 #include "call/rtp_packet_sink_interface.h"
 #include "call/syncable.h"
 #include "modules/audio_coding/include/audio_coding_module_typedefs.h"
+#include "modules/rtp_rtcp/include/rtp_rtcp_defines.h"
 #include "modules/rtp_rtcp/source/source_tracker.h"
 #include "system_wrappers/include/clock.h"
 
-- 
2.34.1

